lib/api/controllers/controller.js

```diff
@@ -82,8 +82,11 @@ class BaseController {
       const booleanValue = flagValue !== undefined ? true : false;
 
       _.set(request, flagPath, booleanValue);
+
+      return booleanValue;
     }
 
+    return Boolean(flagValue);
```

Comment:
this is not `ensureBooleanFlag` anymore, since you're now retrieving the value

another name must now be found, for instance `tryGetBoolean`?

also, the jsdoc is now outdated